.circleci/config.yml

```diff
@@ -150,6 +173,14 @@ workflows:
           requires:
             - build_and_test
 
+      - test_deprecation_warnings:
+          matrix:
+            parameters:
+              version:
+                - "3.9"
```

Comment:
I think this should be the latest aka most recent supported version. Otherwise issues are detected when it's already too late.

Reply:
I have changed all to 3.12